components/StoryList/index.js

```diff
@@ -47,16 +46,18 @@ const StoryCreationDate = styled.p`
   margin: 0;
 `;
 
-function StoryList() {
+function StoryList({ stories }) {
   return (
     <StoryListContainer>
       <Header>My Stories</Header>
       {stories.map((story) => (
-        <StoryListCard key={story.id}>
-          <StoryCoverImage src={story.coverImage} alt={story.title} />
-          <StoryTitle>{story.title}</StoryTitle>
-          <StoryCreationDate>{story.dateCreated}</StoryCreationDate>
-        </StoryListCard>
+        <Link key={story.id} href={`/stories/${story.id}`}>
+          <StoryListCard>
+            <StoryCoverImage src={story.coverImage} alt={story.title} />
+            <StoryTitle>{story.title}</StoryTitle>
+            <StoryCreationDate>{story.dateCreated}</StoryCreationDate>
+          </StoryListCard>
+        </Link>
```

Comment:
This is an info post for further reading for after the bootcamp:

```
        <Link key={story.id} href={`/stories/${story.id}`}>
          <StoryListCard>
            <StoryCoverImage src={story.coverImage} alt={story.title} />
            <StoryTitle>{story.title}</StoryTitle>
            <StoryCreationDate>{story.dateCreated}</StoryCreationDate>
          </StoryListCard>
        </Link>
```

This is a more advanced topic, but I decided to remark this part: I would not recommend to wrap your `StoryListCard` with a `Link` to make the whole `Card` clickable as a screenreader would read out the whole content of each card as link text. But the user does not get a info like "see story details" or something like that. [See this link for more information about the topic.](https://adrianroselli.com/2020/02/block-links-cards-clickable-regions-etc.html) You do not need to change that as the html itself is ok. But if you want to add another link inside the StoryListCard, you need to change the current structure.